chevah/compat
# [#695] Update for latest Twisted.

Scope
=====

Fixes #695 

This is here to support latest Twisted version.


Changes
=======

Fix testing of Twisted reactor.

As a drive-by update the python version.

Docker tests for Amazon LInux 2 was removed due to glibc issues

Testing for macOS is now only on M1... as we no longer have x64 Python wheel packages for macos AMD64

How to try and test the changes
===============================

reviewers: @dumol 

More of a FYI for the environments for which we run tests for compat.

This is to make sure the tests in compat are compatible with the kind of testing done in Pythia
